### PR TITLE
feat: accomplishment cards visible in events

### PR DESCRIPTION
**Pull request by Nick42791**
accomplishment cards are visible when the clientViewHandler is set to:
- INFLUENCES SELECT
- INFLUENCES DRAW

### CODE REVIEW

File: client/src/components/game/phases/Events.vue [hunk 1]
```diff
@@ -30,6 +33,24 @@
           </div>
         </div>
       </div>
+      <div class="active-accomplishments col-3" 
+        v-if="currentEvent.clientViewHandler == 'INFLUENCES_SELECT' ||
```
Comment (alee) [inline client/src/components/game/phases/Events.vue:37]: Could we push this conditional logic into the `Events` Vue class instead? Like a getter `shouldShowAccomplishments` or something along those lines?

File: client/src/components/game/phases/Events.vue [hunk 2]
```diff
@@ -18,7 +18,10 @@
           </div>
         </div>
       </div>
-      <div class="active-events col-8 tour-active-events">
+      <div class="active-events tour-active-events"
+        v-bind="{class: currentEvent.clientViewHandler == 'INFLUENCES_SELECT' ||
```
Comment (alee) [inline client/src/components/game/phases/Events.vue:22]: If we push the conditional logic into a function `shouldShowAccomplishments`, we can reuse it to make the picking the CSS class selection a little simpler. Also you'll only need to update it in one place if it ever changes.

File: server/tests/rooms/game/index.test.ts [hunk 3]
```diff
@@ -199,4 +201,18 @@ describe('inverting pending inventory', () => {
     expect(g.players['Curator'].pendingInvestments.culture).toBe(-3);
     expect(g.players['Curator'].inventory.culture).toBe(3);
   });
-});
\ No newline at end of file
+});
+
+describe('out of commission event', () => {
+  const g = new GameState(mockGameStateInitOpts(x => x, () => 10));
+  const outOfCommissionEntreprenuer = getAllMarsEvents()[7][0];
```
Comment (cpritcha) [inline server/tests/rooms/game/index.test.ts:208]: grab the `outOfCommissionEntrepreneur` directly instead of relying on its position in array

Comment (cpritcha) [reply server/tests/rooms/game/index.test.ts:208]: this code is already fixed in another pull request how just removing the changes to this file?

Comment (Nick42791) [reply server/tests/rooms/game/index.test.ts:208]: I really don't know why it's in here at all.
I'll make that change. Thanks!

**Comment by chrstngyn** [comment]
yup, looks good!

**Pull request merged**